2025-08-08 02:56:11 GMT [9]: [3-1] user=,db=,app=,client= LOG:  starting PostgreSQL 17.5 (Debian 17.5-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
2025-08-08 02:56:11 GMT [9]: [4-1] user=,db=,app=,client= LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-08-08 02:56:11 GMT [9]: [5-1] user=,db=,app=,client= LOG:  listening on IPv6 address "::", port 5432
2025-08-08 02:56:11 GMT [9]: [6-1] user=,db=,app=,client= LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-08-08 02:56:11 GMT [15]: [1-1] user=,db=,app=,client= LOG:  database system was interrupted; last known up at 2025-08-08 02:38:08 GMT
2025-08-08 02:56:11 GMT [15]: [2-1] user=,db=,app=,client= LOG:  database system was not properly shut down; automatic recovery in progress
2025-08-08 02:56:11 GMT [15]: [3-1] user=,db=,app=,client= LOG:  redo starts at 0/A000530
2025-08-08 02:56:11 GMT [15]: [4-1] user=,db=,app=,client= LOG:  invalid record length at 0/A000638: expected at least 24, got 0
2025-08-08 02:56:11 GMT [15]: [5-1] user=,db=,app=,client= LOG:  redo done at 0/A000600 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-08-08 02:56:11 GMT [13]: [1-1] user=,db=,app=,client= LOG:  checkpoint starting: end-of-recovery immediate wait
2025-08-08 02:56:11 GMT [13]: [2-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.008 s, total=0.014 s; sync files=2, longest=0.006 s, average=0.004 s; distance=0 kB, estimate=0 kB; lsn=0/A000638, redo lsn=0/A000638
2025-08-08 02:56:11 GMT [9]: [7-1] user=,db=,app=,client= LOG:  database system is ready to accept connections
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:56:11 GMT [18]: [1-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:56:11 GMT [18]: [2-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:56:12 GMT [18]: [3-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:56:12 GMT [18]: [4-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 02:56:13 GMT [25]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=::1 LOG:  connection received: host=::1 port=47178
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:56:13 GMT [18]: [5-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:56:13 GMT [18]: [6-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 02:56:13 GMT [18]: [7-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 02:56:14 GMT [159]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=46700
2025-08-08 02:56:14 GMT [159]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 02:56:14 GMT [159]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 02:56:41 GMT [210]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:56:41 GMT [210]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:56:41 GMT [210]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:56:41 GMT [210]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 02:56:57 GMT [211]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection received: host=172.20.0.11 port=46772
2025-08-08 02:56:57 GMT [211]: [2-1] user=replicator,db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection authenticated: identity="replicator" method=md5 (/var/lib/postgresql/data/pg_hba.conf:18)
2025-08-08 02:56:57 GMT [211]: [3-1] user=replicator,db=[unknown],app=[unknown],client=172.20.0.11 LOG:  replication connection authorized: user=replicator application_name=efm_cluster_standby
2025-08-08 02:57:00 GMT [219]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection received: host=172.20.0.11 port=46788
2025-08-08 02:57:00 GMT [219]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.11 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 02:57:00 GMT [219]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.11 LOG:  connection authorized: user=efm database=postgres
2025-08-08 02:57:00 GMT [219]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.11 LOG:  disconnection: session time: 0:00:00.003 user=efm database=postgres host=172.20.0.11 port=46788
2025-08-08 02:57:11 GMT [229]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:57:11 GMT [229]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:57:11 GMT [229]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:57:11 GMT [229]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 02:57:13 GMT [237]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.12 LOG:  connection received: host=172.20.0.12 port=57080
2025-08-08 02:57:13 GMT [237]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.12 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 02:57:13 GMT [237]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.12 LOG:  connection authorized: user=efm database=postgres
2025-08-08 02:57:13 GMT [237]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.12 LOG:  disconnection: session time: 0:00:00.030 user=efm database=postgres host=172.20.0.12 port=57080
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:57:13 GMT [18]: [8-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:57:13 GMT [18]: [9-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:57:14 GMT [18]: [10-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:57:14 GMT [18]: [11-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:57:15 GMT [18]: [12-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:57:15 GMT [18]: [13-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 02:57:15 GMT [18]: [14-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 02:57:41 GMT [251]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:57:41 GMT [251]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:57:41 GMT [251]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:57:41 GMT [251]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 02:57:57 GMT [159]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:43.187 user=efm database=postgres host=172.20.0.10 port=46700
2025-08-08 02:57:57 GMT [259]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=51646
2025-08-08 02:57:57 GMT [259]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 02:57:57 GMT [259]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 02:58:08 GMT [261]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:58:08 GMT [261]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:58:08 GMT [261]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-08 02:58:11 GMT [270]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:58:11 GMT [270]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:58:11 GMT [270]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:58:11 GMT [270]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 02:58:15 GMT [261]: [4-1] user=postgres,db=postgres,app=psql,client=[local] NOTICE:  relation "test_replication" already exists, skipping
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:58:15 GMT [18]: [15-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:58:15 GMT [18]: [16-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:58:16 GMT [18]: [17-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:58:16 GMT [18]: [18-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:58:17 GMT [18]: [19-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:58:17 GMT [18]: [20-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 02:58:17 GMT [18]: [21-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 02:58:41 GMT [294]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:58:41 GMT [294]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:58:41 GMT [294]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:58:41 GMT [294]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 02:59:12 GMT [305]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:59:12 GMT [305]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:59:12 GMT [305]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:59:12 GMT [305]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:59:17 GMT [18]: [22-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:59:17 GMT [18]: [23-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:59:18 GMT [18]: [24-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:59:18 GMT [18]: [25-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 02:59:19 GMT [18]: [26-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 02:59:19 GMT [18]: [27-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 02:59:19 GMT [18]: [28-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 02:59:37 GMT [259]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.996 user=efm database=postgres host=172.20.0.10 port=51646
2025-08-08 02:59:37 GMT [315]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=45470
2025-08-08 02:59:37 GMT [315]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 02:59:37 GMT [315]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 02:59:42 GMT [325]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 02:59:42 GMT [325]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 02:59:42 GMT [325]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 02:59:42 GMT [325]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:00:12 GMT [334]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:00:12 GMT [334]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:00:12 GMT [334]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:00:12 GMT [334]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:00:19 GMT [18]: [29-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:00:19 GMT [18]: [30-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:00:20 GMT [18]: [31-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:00:20 GMT [18]: [32-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:00:21 GMT [18]: [33-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:00:21 GMT [18]: [34-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:00:21 GMT [18]: [35-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:00:42 GMT [353]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:00:42 GMT [353]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:00:42 GMT [353]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:00:42 GMT [353]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
2025-08-08 03:01:11 GMT [13]: [3-1] user=,db=,app=,client= LOG:  checkpoint starting: time
2025-08-08 03:01:12 GMT [13]: [4-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.315 s, sync=0.009 s, total=0.342 s; sync files=4, longest=0.003 s, average=0.003 s; distance=1 kB, estimate=1 kB; lsn=0/A000D48, redo lsn=0/A000CF0
2025-08-08 03:01:12 GMT [363]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:01:12 GMT [363]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:01:12 GMT [363]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:01:12 GMT [363]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:01:17 GMT [315]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.994 user=efm database=postgres host=172.20.0.10 port=45470
2025-08-08 03:01:17 GMT [364]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=42992
2025-08-08 03:01:17 GMT [364]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:01:17 GMT [364]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:01:22 GMT [18]: [36-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:01:22 GMT [18]: [37-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:01:23 GMT [18]: [38-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:01:23 GMT [18]: [39-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:01:24 GMT [18]: [40-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:01:24 GMT [18]: [41-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:01:24 GMT [18]: [42-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:01:42 GMT [378]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:01:42 GMT [378]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:01:42 GMT [378]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:01:42 GMT [378]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:02:12 GMT [388]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:02:12 GMT [388]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:02:12 GMT [388]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:02:12 GMT [388]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:02:24 GMT [18]: [43-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:02:24 GMT [18]: [44-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:02:25 GMT [18]: [45-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:02:25 GMT [18]: [46-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:02:26 GMT [18]: [47-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:02:26 GMT [18]: [48-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:02:26 GMT [18]: [49-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:02:42 GMT [405]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:02:42 GMT [405]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:02:42 GMT [405]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:02:42 GMT [405]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:02:57 GMT [364]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.990 user=efm database=postgres host=172.20.0.10 port=42992
2025-08-08 03:02:57 GMT [406]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=59516
2025-08-08 03:02:57 GMT [406]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:02:57 GMT [406]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:03:12 GMT [414]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:03:12 GMT [414]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:03:12 GMT [414]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:03:12 GMT [414]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:03:26 GMT [18]: [50-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:03:26 GMT [18]: [51-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:03:27 GMT [18]: [52-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:03:27 GMT [18]: [53-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:03:28 GMT [18]: [54-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:03:28 GMT [18]: [55-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:03:28 GMT [18]: [56-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:03:42 GMT [429]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:03:42 GMT [429]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:03:42 GMT [429]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:03:42 GMT [429]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.007 user=postgres database=postgres host=[local]
2025-08-08 03:04:12 GMT [438]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:04:12 GMT [438]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:04:12 GMT [438]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:04:12 GMT [438]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:04:28 GMT [18]: [57-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:04:28 GMT [18]: [58-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:04:29 GMT [18]: [59-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:04:29 GMT [18]: [60-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:04:30 GMT [18]: [61-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:04:30 GMT [18]: [62-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:04:30 GMT [18]: [63-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:04:37 GMT [406]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.993 user=efm database=postgres host=172.20.0.10 port=59516
2025-08-08 03:04:37 GMT [448]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=48326
2025-08-08 03:04:37 GMT [448]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:04:37 GMT [448]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:04:42 GMT [456]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:04:42 GMT [456]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:04:42 GMT [456]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:04:42 GMT [456]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:05:12 GMT [467]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:05:12 GMT [467]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:05:12 GMT [467]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:05:12 GMT [467]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:05:30 GMT [18]: [64-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:05:30 GMT [18]: [65-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:05:31 GMT [18]: [66-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:05:31 GMT [18]: [67-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:05:32 GMT [18]: [68-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:05:32 GMT [18]: [69-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:05:32 GMT [18]: [70-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:05:42 GMT [481]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:05:42 GMT [481]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:05:42 GMT [481]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:05:42 GMT [481]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:06:12 GMT [492]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:06:12 GMT [492]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:06:12 GMT [492]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:06:12 GMT [492]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.004 user=postgres database=postgres host=[local]
2025-08-08 03:06:17 GMT [448]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.993 user=efm database=postgres host=172.20.0.10 port=48326
2025-08-08 03:06:17 GMT [493]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=41946
2025-08-08 03:06:17 GMT [493]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:06:17 GMT [493]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:06:32 GMT [18]: [71-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:06:32 GMT [18]: [72-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:06:33 GMT [18]: [73-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:06:33 GMT [18]: [74-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:06:34 GMT [18]: [75-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:06:34 GMT [18]: [76-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:06:34 GMT [18]: [77-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:06:43 GMT [508]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:06:43 GMT [508]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:06:43 GMT [508]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:06:43 GMT [508]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:07:13 GMT [517]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:07:13 GMT [517]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:07:13 GMT [517]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:07:13 GMT [517]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:07:34 GMT [18]: [78-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:07:34 GMT [18]: [79-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:07:35 GMT [18]: [80-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:07:35 GMT [18]: [81-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:07:36 GMT [18]: [82-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:07:36 GMT [18]: [83-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:07:36 GMT [18]: [84-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:07:43 GMT [533]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:07:43 GMT [533]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:07:43 GMT [533]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:07:43 GMT [533]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:07:57 GMT [493]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.989 user=efm database=postgres host=172.20.0.10 port=41946
2025-08-08 03:07:57 GMT [534]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=44008
2025-08-08 03:07:57 GMT [534]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:07:57 GMT [534]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:08:13 GMT [545]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:08:13 GMT [545]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:08:13 GMT [545]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:08:13 GMT [545]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:08:36 GMT [18]: [85-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:08:36 GMT [18]: [86-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:08:37 GMT [18]: [87-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:08:37 GMT [18]: [88-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:08:38 GMT [18]: [89-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:08:38 GMT [18]: [90-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:08:38 GMT [18]: [91-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:08:43 GMT [561]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:08:43 GMT [561]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:08:43 GMT [561]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:08:43 GMT [561]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:09:13 GMT [571]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:09:13 GMT [571]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:09:13 GMT [571]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:09:13 GMT [571]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:09:37 GMT [534]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.985 user=efm database=postgres host=172.20.0.10 port=44008
2025-08-08 03:09:37 GMT [573]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=50916
2025-08-08 03:09:37 GMT [573]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:09:37 GMT [573]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:09:38 GMT [18]: [92-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:09:38 GMT [18]: [93-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:09:39 GMT [18]: [94-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:09:39 GMT [18]: [95-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:09:40 GMT [18]: [96-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:09:40 GMT [18]: [97-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:09:40 GMT [18]: [98-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:09:43 GMT [587]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:09:43 GMT [587]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:09:43 GMT [587]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:09:43 GMT [587]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:10:13 GMT [598]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:10:13 GMT [598]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:10:13 GMT [598]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:10:13 GMT [598]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:10:40 GMT [18]: [99-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:10:40 GMT [18]: [100-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:10:41 GMT [18]: [101-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:10:41 GMT [18]: [102-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:10:42 GMT [18]: [103-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:10:42 GMT [18]: [104-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:10:42 GMT [18]: [105-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:10:43 GMT [612]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:10:43 GMT [612]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:10:43 GMT [612]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:10:43 GMT [612]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:11:13 GMT [621]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:11:13 GMT [621]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:11:13 GMT [621]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:11:13 GMT [621]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:11:17 GMT [573]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.986 user=efm database=postgres host=172.20.0.10 port=50916
2025-08-08 03:11:17 GMT [622]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=48414
2025-08-08 03:11:17 GMT [622]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:11:17 GMT [622]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:11:42 GMT [18]: [106-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:11:42 GMT [18]: [107-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:11:43 GMT [18]: [108-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:11:43 GMT [18]: [109-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:11:43 GMT [636]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:11:43 GMT [636]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:11:43 GMT [636]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:11:43 GMT [636]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:11:44 GMT [18]: [110-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:11:44 GMT [18]: [111-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:11:44 GMT [18]: [112-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:12:13 GMT [647]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:12:13 GMT [647]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:12:13 GMT [647]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:12:13 GMT [647]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:12:43 GMT [659]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:12:43 GMT [659]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:12:43 GMT [659]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:12:43 GMT [659]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:12:44 GMT [18]: [113-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:12:44 GMT [18]: [114-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:12:45 GMT [18]: [115-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:12:45 GMT [18]: [116-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:12:46 GMT [18]: [117-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:12:46 GMT [18]: [118-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:12:46 GMT [18]: [119-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:12:57 GMT [622]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.994 user=efm database=postgres host=172.20.0.10 port=48414
2025-08-08 03:12:57 GMT [666]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=59618
2025-08-08 03:12:57 GMT [666]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:12:57 GMT [666]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:13:13 GMT [677]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:13:13 GMT [677]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:13:13 GMT [677]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:13:13 GMT [677]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:13:43 GMT [685]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:13:43 GMT [685]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:13:43 GMT [685]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:13:43 GMT [685]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:13:46 GMT [18]: [120-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:13:46 GMT [18]: [121-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:13:47 GMT [18]: [122-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:13:47 GMT [18]: [123-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:13:48 GMT [18]: [124-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:13:48 GMT [18]: [125-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:13:48 GMT [18]: [126-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:14:14 GMT [702]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:14:14 GMT [702]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:14:14 GMT [702]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:14:14 GMT [702]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:14:37 GMT [666]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.997 user=efm database=postgres host=172.20.0.10 port=59618
2025-08-08 03:14:37 GMT [703]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=59770
2025-08-08 03:14:37 GMT [703]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:14:37 GMT [703]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:14:44 GMT [711]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:14:44 GMT [711]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:14:44 GMT [711]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:14:44 GMT [711]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:14:48 GMT [18]: [127-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:14:48 GMT [18]: [128-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:14:49 GMT [18]: [129-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:14:49 GMT [18]: [130-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:14:50 GMT [18]: [131-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:14:50 GMT [18]: [132-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:14:50 GMT [18]: [133-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:15:14 GMT [728]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:15:14 GMT [728]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:15:14 GMT [728]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:15:14 GMT [728]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:15:44 GMT [737]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:15:44 GMT [737]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:15:44 GMT [737]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:15:44 GMT [737]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:15:50 GMT [18]: [134-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:15:50 GMT [18]: [135-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:15:51 GMT [18]: [136-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:15:51 GMT [18]: [137-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:15:52 GMT [18]: [138-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:15:52 GMT [18]: [139-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:15:52 GMT [18]: [140-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:16:14 GMT [753]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:16:14 GMT [753]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:16:14 GMT [753]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:16:14 GMT [753]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:16:17 GMT [703]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.984 user=efm database=postgres host=172.20.0.10 port=59770
2025-08-08 03:16:17 GMT [757]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=39642
2025-08-08 03:16:17 GMT [757]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:16:17 GMT [757]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:16:44 GMT [765]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:16:44 GMT [765]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:16:44 GMT [765]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:16:44 GMT [765]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:16:52 GMT [18]: [141-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:16:52 GMT [18]: [142-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:16:53 GMT [18]: [143-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:16:53 GMT [18]: [144-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:16:54 GMT [18]: [145-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:16:54 GMT [18]: [146-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:16:54 GMT [18]: [147-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:17:14 GMT [781]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:17:14 GMT [781]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:17:14 GMT [781]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:17:14 GMT [781]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:17:44 GMT [790]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:17:44 GMT [790]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:17:44 GMT [790]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:17:44 GMT [790]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:17:54 GMT [18]: [148-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:17:54 GMT [18]: [149-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:17:55 GMT [18]: [150-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:17:55 GMT [18]: [151-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:17:56 GMT [18]: [152-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:17:56 GMT [18]: [153-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:17:56 GMT [18]: [154-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:17:57 GMT [757]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.991 user=efm database=postgres host=172.20.0.10 port=39642
2025-08-08 03:17:57 GMT [798]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=47274
2025-08-08 03:17:57 GMT [798]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:17:57 GMT [798]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:18:14 GMT [807]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:18:14 GMT [807]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:18:14 GMT [807]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:18:14 GMT [807]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:18:44 GMT [815]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:18:44 GMT [815]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:18:44 GMT [815]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:18:44 GMT [815]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:18:56 GMT [18]: [155-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:18:56 GMT [18]: [156-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:18:57 GMT [18]: [157-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:18:57 GMT [18]: [158-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:18:58 GMT [18]: [159-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:18:58 GMT [18]: [160-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:18:58 GMT [18]: [161-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:19:14 GMT [830]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:19:14 GMT [830]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:19:14 GMT [830]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:19:14 GMT [830]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:19:37 GMT [798]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.996 user=efm database=postgres host=172.20.0.10 port=47274
2025-08-08 03:19:37 GMT [831]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=46636
2025-08-08 03:19:37 GMT [831]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:19:37 GMT [831]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:19:44 GMT [839]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:19:44 GMT [839]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:19:44 GMT [839]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:19:44 GMT [839]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:19:59 GMT [18]: [162-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:19:59 GMT [18]: [163-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:20:00 GMT [18]: [164-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:20:00 GMT [18]: [165-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:20:01 GMT [18]: [166-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:20:01 GMT [18]: [167-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:20:01 GMT [18]: [168-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:20:14 GMT [855]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:20:14 GMT [855]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:20:14 GMT [855]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:20:14 GMT [855]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:20:44 GMT [865]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:20:44 GMT [865]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:20:44 GMT [865]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:20:44 GMT [865]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:21:01 GMT [18]: [169-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:21:01 GMT [18]: [170-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:21:02 GMT [18]: [171-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:21:02 GMT [18]: [172-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:21:03 GMT [18]: [173-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:21:03 GMT [18]: [174-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:21:03 GMT [18]: [175-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:21:15 GMT [882]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:21:15 GMT [882]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:21:15 GMT [882]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:21:15 GMT [882]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:21:17 GMT [831]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.991 user=efm database=postgres host=172.20.0.10 port=46636
2025-08-08 03:21:17 GMT [883]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=60370
2025-08-08 03:21:17 GMT [883]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:21:17 GMT [883]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:21:45 GMT [891]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:21:45 GMT [891]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:21:45 GMT [891]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:21:45 GMT [891]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:22:03 GMT [18]: [176-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:22:03 GMT [18]: [177-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:22:04 GMT [18]: [178-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:22:04 GMT [18]: [179-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:22:05 GMT [18]: [180-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:22:05 GMT [18]: [181-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:22:05 GMT [18]: [182-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:22:15 GMT [906]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:22:15 GMT [906]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:22:15 GMT [906]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:22:15 GMT [906]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:22:45 GMT [916]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:22:45 GMT [916]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:22:45 GMT [916]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:22:45 GMT [916]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
2025-08-08 03:22:57 GMT [883]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.986 user=efm database=postgres host=172.20.0.10 port=60370
2025-08-08 03:22:57 GMT [918]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=36968
2025-08-08 03:22:57 GMT [918]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:22:57 GMT [918]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:23:01 GMT [261]: [5-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:24:53.418 user=postgres database=postgres host=[local]
2025-08-08 03:23:03 GMT [1021]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=36980
2025-08-08 03:23:03 GMT [1021]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:23:03 GMT [1021]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:23:03 GMT [1021]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:00:00.044 user=efm database=postgres host=172.20.0.10 port=36980
2025-08-08 03:23:03 GMT [1023]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=36992
2025-08-08 03:23:03 GMT [1023]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:23:03 GMT [1023]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:23:03 GMT [1023]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:00:00.002 user=efm database=postgres host=172.20.0.10 port=36992
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:23:05 GMT [18]: [183-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:23:05 GMT [18]: [184-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:23:06 GMT [18]: [185-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:23:06 GMT [18]: [186-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:23:07 GMT [18]: [187-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:23:07 GMT [18]: [188-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:23:07 GMT [18]: [189-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:23:09 GMT [1033]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection received: host=172.20.0.11 port=46372
2025-08-08 03:23:09 GMT [1033]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.11 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:23:09 GMT [1033]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.11 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:23:09 GMT [1033]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.11 LOG:  disconnection: session time: 0:00:00.038 user=efm database=postgres host=172.20.0.11 port=46372
2025-08-08 03:23:09 GMT [1034]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.11 LOG:  connection received: host=172.20.0.11 port=46382
2025-08-08 03:23:09 GMT [1034]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.11 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:23:09 GMT [1034]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.11 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:23:09 GMT [1034]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.11 LOG:  disconnection: session time: 0:00:00.002 user=efm database=postgres host=172.20.0.11 port=46382
2025-08-08 03:23:15 GMT [1043]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:23:15 GMT [1043]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:23:15 GMT [1043]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:23:15 GMT [1043]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:23:45 GMT [1052]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:23:45 GMT [1052]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:23:45 GMT [1052]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:23:45 GMT [1052]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:24:07 GMT [18]: [190-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:24:07 GMT [18]: [191-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:24:08 GMT [18]: [192-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:24:08 GMT [18]: [193-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:24:09 GMT [18]: [194-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:24:09 GMT [18]: [195-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:24:09 GMT [18]: [196-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:24:15 GMT [1067]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:24:15 GMT [1067]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:24:15 GMT [1067]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:24:15 GMT [1067]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:24:37 GMT [918]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.986 user=efm database=postgres host=172.20.0.10 port=36968
2025-08-08 03:24:37 GMT [1068]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=36204
2025-08-08 03:24:37 GMT [1068]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:24:37 GMT [1068]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:24:45 GMT [1075]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:24:45 GMT [1075]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:24:45 GMT [1075]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:24:45 GMT [1075]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:25:09 GMT [18]: [197-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:25:09 GMT [18]: [198-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:25:10 GMT [18]: [199-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:25:10 GMT [18]: [200-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:25:11 GMT [18]: [201-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:25:11 GMT [18]: [202-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:25:11 GMT [18]: [203-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:25:15 GMT [1090]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:25:15 GMT [1090]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:25:15 GMT [1090]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:25:15 GMT [1090]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-08 03:25:45 GMT [1100]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:25:45 GMT [1100]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:25:45 GMT [1100]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:25:45 GMT [1100]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:26:11 GMT [18]: [204-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:26:11 GMT [18]: [205-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:26:12 GMT [18]: [206-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:26:12 GMT [18]: [207-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:26:13 GMT [18]: [208-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:26:13 GMT [18]: [209-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:26:13 GMT [18]: [210-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:26:15 GMT [1117]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:26:15 GMT [1117]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:26:15 GMT [1117]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:26:15 GMT [1117]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:26:17 GMT [1068]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.991 user=efm database=postgres host=172.20.0.10 port=36204
2025-08-08 03:26:17 GMT [1120]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=36614
2025-08-08 03:26:17 GMT [1120]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:26:17 GMT [1120]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:26:45 GMT [1128]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:26:45 GMT [1128]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:26:45 GMT [1128]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:26:45 GMT [1128]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:27:13 GMT [18]: [211-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:27:13 GMT [18]: [212-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:27:14 GMT [18]: [213-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:27:14 GMT [18]: [214-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:27:15 GMT [18]: [215-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:27:15 GMT [18]: [216-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:27:15 GMT [18]: [217-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:27:15 GMT [1145]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:27:15 GMT [1145]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:27:15 GMT [1145]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:27:15 GMT [1145]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:27:45 GMT [1153]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:27:45 GMT [1153]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:27:45 GMT [1153]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:27:45 GMT [1153]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-08 03:27:57 GMT [1120]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.996 user=efm database=postgres host=172.20.0.10 port=36614
2025-08-08 03:27:57 GMT [1154]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=53896
2025-08-08 03:27:57 GMT [1154]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:27:57 GMT [1154]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:28:15 GMT [18]: [218-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:28:15 GMT [18]: [219-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:28:16 GMT [1165]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:28:16 GMT [1165]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:28:16 GMT [1165]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:28:16 GMT [1165]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:28:16 GMT [18]: [220-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:28:16 GMT [18]: [221-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:28:17 GMT [18]: [222-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:28:17 GMT [18]: [223-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:28:17 GMT [18]: [224-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:28:46 GMT [1179]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:28:46 GMT [1179]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:28:46 GMT [1179]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:28:46 GMT [1179]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
2025-08-08 03:29:16 GMT [1188]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:29:16 GMT [1188]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:29:16 GMT [1188]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:29:16 GMT [1188]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:29:17 GMT [18]: [225-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:29:17 GMT [18]: [226-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:29:18 GMT [18]: [227-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:29:18 GMT [18]: [228-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
cp: cannot create regular file '/var/lib/postgresql/archive/00000003.history': No such file or directory
2025-08-08 03:29:19 GMT [18]: [229-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-08 03:29:19 GMT [18]: [230-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/00000003.history && cp pg_wal/00000003.history /var/lib/postgresql/archive/00000003.history
2025-08-08 03:29:19 GMT [18]: [231-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "00000003.history" failed too many times, will try again later
2025-08-08 03:29:37 GMT [1154]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:01:39.994 user=efm database=postgres host=172.20.0.10 port=53896
2025-08-08 03:29:37 GMT [1195]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=39780
2025-08-08 03:29:37 GMT [1195]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-08 03:29:37 GMT [1195]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-08 03:29:46 GMT [1203]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-08 03:29:46 GMT [1203]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-08 03:29:46 GMT [1203]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-08 03:29:46 GMT [1203]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
